aidants_connect_web/templates/footer/faq.html

```diff
@@ -66,18 +66,18 @@ <h2 id="combien-coûte-AC">Combien coûte Aidants Connect ?</h2>
 
     <h2 id="plusieurs-comptes-AC-pour-une-structure-">Est-ce possible d’avoir plusieurs comptes Aidants Connect pour une même structure ?</h2>
     <p>
-      Oui, une structure peut avoir plusieurs aidants professionnels habilités : chaque aidant possède un compte nominatif.
-      Tous les aidants habilités sont sensibilisés à la protection des données personnelles des usagers.</p>
+      Oui, une structure peut avoir plusieurs aidant·es professionnel·les habilité·es : chaque aidant·e possède un compte nominatif.
+      Tou·tes les aidant·es habilité·es sont sensibilisé·es à la protection des données personnelles des usager·ères.</p>
 
-    <h2 id="agir-sur-tous-les-mandats-de-la-structure-">Est-ce que tous les aidants habilités peuvent consulter/agir sur tous les mandats de la structure ?</h2>
+    <h2 id="agir-sur-tous-les-mandats-de-la-structure-">Est-ce que tou·tes les aidant·es habilité·es peuvent consulter/agir sur tous les mandats de la structure ?</h2>
     <p>
-      Les mandats étant signés entre la structure et l’usager, tous les aidants habilités de la structure peuvent aider
-      l’usager sur les périmètres et la durée couverts par le mandat.
+      Les mandats étant signés entre la structure et l’usager·ère, tou·tes les aidant·es habilité·es de la structure peuvent aider
+      l’usager·ère sur les périmètres et la durée couverts par le mandat.
     </p>
 
-    <h2 id="login-site-administratif">L’usager doit-il avoir un identifiant/mot de passe sur un site administratif ?</h2>
+    <h2 id="login-site-administratif">L’usager·ère doit-il avoir un identifiant/mot de passe sur un site administratif ?</h2>
```

Comment:
```suggestion
    <h2 id="login-site-administratif">Est-il nécessaire pour l’usager·ère d'avoir un identifiant/mot de passe sur un site administratif ?</h2>
```

(pour éviter le casse-tête avec `doit-il` 😋)